cmake/configs/default.cmake

```diff
@@ -78,7 +78,7 @@ hunter_default_version(HastyNoise VERSION 0.8.3)
 hunter_default_version(ICU VERSION 63.1-p5)
 hunter_default_version(IF97 VERSION 2.1.2)
 hunter_default_version(Igloo VERSION 1.1.1-hunter)
-hunter_default_version(IlmBase VERSION 2.4.0)
+hunter_default_version(IlmBase VERSION 2.4.0-p0)
```

Comment:
Should we make this `2.4.0-p1` so the OpenEXR and IlmBase packages are from the same archive?

Reply:
Makes sense, done